pom.xml

```diff
@@ -23,11 +23,18 @@
         <version>1.0.9</version>
     </parent>
 
-    <groupId>jakarta.enterprise.concurrent</groupId>
-    <artifactId>jakarta.enterprise.concurrent.parent</artifactId>
+    <groupId>jakarta.concurrent</groupId>
+    <artifactId>jakarta.concurrent.parent</artifactId>
     <version>3.2.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <distributionManagement>
```

Comment:
Like I said previously for api - I don't think this is right.

Reply:
@pzygielo - it sounds like you are more knowledgeable in this area.  Would you want to put together a pull request for how this should be done?

Reply:
Almost all is fine. I'd remove this element in top-level pom only.

Reply:
Does this look correct? [`2fd7300` (#555)](https://github.com/jakartaee/concurrency/pull/555/commits/2fd7300ebab45a9c6b724b42ca556f5e3997cb4b)